x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/plan/logical/command/sys/SysColumnsTests.java

```diff
@@ -501,10 +506,29 @@ public void testSysColumnsWithNullCatalog() throws Exception {
         }, mapping);
     }
 
+    private static void checkOdbcShortTypes(SchemaRowSet r) {
```

Comment:
Personal preference: I like to have the private methods at the bottom of the class and not before they are first used.

Reply:
Thanks, updated.
I've also updated the actual fix, since it wasn't entirely correct regarding pagination.